packages/remix-server-runtime/server.ts

```diff
@@ -110,7 +110,7 @@ async function handleDataRequest({
       match = getRequestMatch(url, matches);
 
       response = await callRouteAction({
-        loadContext,
+        loadContext: loadContext ?? {},
```

Comment:
I would default this as high up as possible. So, in the args definition  line 79) or even higher up. I'm assuming that the consumer of this function doesn't really or shouldn't be instantiating the empty object.

Reply:
Good call. I moved it up to the `requestHandler` to be the default value for the loadContext arg. This definitely did clean it up